rest/config.go

```diff
@@ -265,8 +265,16 @@ type SequenceHashConfig struct {
 	Frequency    *int    `json:"hash_frequency,omitempty"` // Frequency of sequence hashing in changes feeds
 }
 
+type Logging struct {
```

Comment:
Can we rename this UnsupportedLogging for clarity?

Reply:
Actually, do we anticipate more unsupported logging stats beyond StatsLogFrequency?  If not we should just move it to a top level property in UnsupportedServerConfig.

Reply:
Moved up to a top level property in UnsupportedServerConfig.